downloads/downloads.js

```diff
@@ -28,4 +28,13 @@ $(document).ready(function () {
         }
     });
 
+    // Look for hash in URL and open MSW binaries model if available
+    if (document.location.hash) {
+        var data = /^#v(.\..\..)_msw$/.exec(document.location.hash);
```

Comment:
AFAICS this won't work for 2 digit versions, but this is probably not a problem in practice...